src/main/java/com/jme3/math/Quaternion.java

```diff
@@ -803,110 +803,115 @@ public float toAngleAxis(Vector3f axisStore) {
 		return angle;
 	}
 
+	public float angleBetween(Quaternion q2) {
+		float w = this.w * q2.w + this.x * q2.x + this.y * q2.y + this.z * q2.z;
+		float x = this.w * q2.x - this.x * q2.w - this.y * q2.z + this.z * q2.y;
+		float y = this.w * q2.y + this.x * q2.z - this.y * q2.w - this.z * q2.x;
+		float z = this.w * q2.z - this.x * q2.y + this.y * q2.x - this.z * q2.w;
+
+		// compute cosine and sine of the angle between
+		// do so in a numerically stable way
+		float ang = FastMath.atan2(FastMath.sqrt(x * x + y * y + z * z), w);
+
+		return ang;
```

Comment:
Just return `ang` right away
```suggestion
		return FastMath.atan2(FastMath.sqrt(x * x + y * y + z * z), w);
```